openapi/schemas/processes-core/statusInfo.yaml

```diff
@@ -1,42 +1,47 @@
-type: object
-required:
-   - id
-   - status
-   - type
-properties:
-   id:
-      type: string
-   processID:
-      type: string
-   type:
-      type: string
-      example:
-        - process
-        - wps
-        - openeo
-   status:
-      $ref: "statusCode.yaml"
-   message:
-      type: string
-   exception:
-      $ref: "../common-core/exception.yaml"
-   created:
-      type: string
-      format: date-time
-   started:
-      type: string
-      format: date-time
-   finished:
-      type: string
-      format: date-time
-   updated:
-      type: string
-      format: date-time
-   progress:
-      type: integer
-      minimum: 0
-      maximum: 100
-   links:
-      type: array
-      items:
-         $ref: "../common-core/link.yaml"
+allOf:
+  - $ref: descriptionType.yaml
+  - type: object
+    required:
+       - id
+       - status
+       - created
+       - process
```

Comment:
Should it be required or optional?

Reply:
I reverted back to the required fields for OAProc and included text in the descriptions based on the value of the "type" property.  If the value of "type" is "openeo" then both "created" and "process" MUST be present as per the OpenEO API definition.

Reply:
Thanks. 

Generally, I think we can do better to avoid these if/then conditionals as much as possible :-)